blurring_as_a_service/metrics/custom_metrics_calculator.py

```diff
@@ -0,0 +1,500 @@
+import itertools
+import json
+from enum import Enum
+from os import listdir
+from os.path import isfile, join
+from typing import List
+
+import matplotlib.pyplot as plt
+from tqdm import tqdm
+
+from blurring_as_a_service.metrics.metrics_utils import generate_binary_mask
+from blurring_as_a_service.metrics.total_blurred_area import TotalBlurredArea
+from blurring_as_a_service.metrics.yolo_labels_dataset import YoloLabelsDataset
+from blurring_as_a_service.utils.bias_category_mapper import BiasCategoryMapper
+
+
+class Size(Enum):
```

Comment:
Maybe ImageSize is more easy to understand out of context?

Reply:
If you inherit object instead of enum the object is subscriptable and you can avoid the `.value` in `Size.small.value[0]` and just write `Size.small[0]`.

Reply:
Then I would be losing some nice functionalities from Enum, such as iterating through the attributes, which I am doing multiple times. 

I changed it to this for now: 
```

class ImageSize(Enum):
    small = [0, 5000]
    medium = [5000, 10000]
    large = [10000, 1000000]

    def __repr__(self):
        return self.value

    def __getitem__(self, index):
        return self.value[index]
```

Reply:
Good point :)